src/Raven.Server/Utils/Monitoring/DatabaseMetrics.cs

```diff
@@ -0,0 +1,156 @@
+// -----------------------------------------------------------------------
+//  <copyright file="DatabaseMonitoring.cs" company="Hibernating Rhinos LTD">
+//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.Utils.Monitoring
+{
+    public class DatabaseMetrics
+    {
+        public string Name { get; set; }
+        public string DatabaseId { get; set; }
+        public int UptimeInSec { get; set; }
+        public double? TimeSinceLastBackupInSec { get; set; }
+        
+        public DatabaseCounts Counts { get; set; }
+        public DatabaseStatistics Statistics { get; set; } 
+        public DatabaseIndexesMetrics Indexes { get; set; }
+        public DatabaseStorageMetrics Storage { get; set; }
+
+        public DynamicJsonValue ToJson()
+        {
+            return new DynamicJsonValue
+            {
+                [nameof(Name)] = Name,
+                [nameof(DatabaseId)] = DatabaseId,
+                [nameof(UptimeInSec)] = UptimeInSec,
+                [nameof(TimeSinceLastBackupInSec)] = TimeSinceLastBackupInSec,
+                [nameof(Counts)] = Counts.ToJson(),
+                [nameof(Statistics)] = Statistics.ToJson(),
+                [nameof(Indexes)] = Indexes.ToJson(),
+                [nameof(Storage)] = Storage.ToJson()
+            };
+        }
+    }
+
+    public class DatabaseStatistics
+    {
+        public int DocPutsPerSecond { get; set; }
+        public int MapIndexIndexesPerSecond { get; set; }
+        public int MapReduceIndexMappedPerSecond { get; set; }
+        public int MapReduceIndexReducedPerSecond { get; set; }
+        public int RequestsPerSecond { get; set; }
+        public int RequestsCount { get; set; }
+        public int RequestAverageDuration { get; set; }
+
+        public DynamicJsonValue ToJson()
+        {
+            return new DynamicJsonValue
+            {
+                [nameof(DocPutsPerSecond)] = DocPutsPerSecond,
+                [nameof(MapIndexIndexesPerSecond)] = MapIndexIndexesPerSecond,
+                [nameof(MapReduceIndexMappedPerSecond)] = MapReduceIndexMappedPerSecond,
+                [nameof(MapReduceIndexReducedPerSecond)] = MapReduceIndexReducedPerSecond,
+                [nameof(RequestsPerSecond)] = RequestsPerSecond,
+                [nameof(RequestsCount)] = RequestsCount,
+                [nameof(RequestAverageDuration)] = RequestAverageDuration
+            };
+        }
+    }
+
+    public class DatabaseCounts
+    {
+        public long Documents { get; set; }
+        public long Revisions { get; set; }
+        public long Attachments { get; set; }
+        public long UniqueAttachments { get; set; }
+        public long Alerts { get; set; }
+        public int Rehabs { get; set; }
```

Comment:
What is this?

Reply:
it matches this:

Database Stats | Number of rehabs | 1.3.6.1.4.1.45751.1.1.5.2.X.1.14